### ISSUE TITLE
Set up F1-score measurement tool and record baseline [System Accuracy]

### ISSUE THREAD

**Issue by celina-ayl** [renaa-celina-ml-tactic-tradeoffs-thesis/celestial-bodies-detection-thesis#4]
Acceptance criteria:

- [ ] Implement a system accuracy measurement utility to calculate the F1-score, precision, and recall, using the library sklearn.metrics, this should be calculated after a full training run is complete

- [ ] F1-score is calculated as: 2 * (precision * recall) / (precision + recall)

- [ ] The measurement is repeated 3 times, with a full environment restart between each run

- [ ] An average F1-score, precision, and recall is computed from the 3 readings

- [ ] Baseline results are stored in a measurement log containing: each individual reading, the computed average
